docs/sources/datasources/azuremonitor/template-variables.md

```diff
@@ -62,3 +62,66 @@ Perf
 | summarize avg(CounterValue) by bin(TimeGenerated, $__interval), Computer
 | order by TimeGenerated asc
 ```
+
+## Limitations
+
+As of Grafana 9.0, a resource URI is constructed to identify resources using the resource picker. If a dashboard had been created prior to Grafana 9.0 then any queries utilising the prior resource picking mechanism will be migrated to resource URIs.
+
+This presents an issue as some resource types make use of nested namespaces and resource names e.g:
+`Microsoft.Storage/tableServices` and `storageAccount/default` or `Microsoft.Sql/servers/databases` and `serverName/databaseName`. There are possible cases where template variables cannot be used as a malformed resource URI may be constructed.
+
+### Supported cases
+
+1. Standard namespaces and resource names.
```

Comment:
```suggestion
#### Standard namespaces and resource names
```

Subheadings would be more appropriate, easier to markup, and easier to anchor-link to, than ordered (or unordered) list items here.